.github/workflows/advanced.yml

```diff
@@ -72,6 +72,9 @@ env:
   SOURCE_PATH: ${{ github.workspace }}/code
   ARTIFACT_PATH: ${{ github.workspace }}/artifact
 
+  GITHUB_EVENT_PULL_REQUEST_BASE_REF: ${{ github.event.pull_request.base.ref }}
```

Comment:
I'd call it just `BASE_REF`  and `HEAD_REF` the `GITHUB_EVENT_PULL_REQUEST_` prefix is just a noise.

Reply:
Done